public/app/features/dashboard-scene/pages/PublicDashboardScenePage.tsx

```diff
@@ -99,16 +100,42 @@ function PublicDashboardSceneRenderer({ model }: SceneComponentProps<DashboardSc
           </Stack>
         )}
       </div>
-      <div className={styles.body}>
-        <bodyToRender.Component model={bodyToRender} />
-      </div>
+      <NativeScrollbar divId="page-scrollbar" autoHeightMin={'100%'}>
+        <div className={styles.pageContainer}>
+          <div className={styles.canvasContent}>
+            <bodyToRender.Component model={bodyToRender} />
+          </div>
+        </div>
+      </NativeScrollbar>
       <PublicDashboardFooter />
```

Comment:
I am confused why this needed? I am testing public dashboards no in latest main and see no issues

Reply:
think a simple "    display: flex;"  on the body class fixes it, ash will open a new PR